automation_tools/__init__.py

```diff
@@ -1881,6 +1883,11 @@ def downstream_install(admin_password=None, run_katello_installer=True):
         print('The BASE_URL environment variable should be defined')
         sys.exit(1)
 
+    maintain_url = os.environ.get('MAINTAIN_BASE_URL')
```

Comment:
@pgagne I think we don't need to introduce new variable here. We can simply overwrite MAINTAIN_REPO variable. See my comment at https://github.com/SatelliteQE/robottelo-ci/pull/1570/files#r320761803